src/main/java/ch/wisv/events/model/Event.java

```diff
@@ -0,0 +1,135 @@
+package ch.wisv.events.model;
+
+import com.google.common.base.MoreObjects;
+
+import javax.persistence.*;
```

Comment:
Do we want this btw? You removed something similar in 06d3f43458dab9211336b1bb6244d984ca1d6e8a...